File: WineriesApp/src/WineriesApp.Services/Models/WinerySearchInfo.cs
namespace WineriesApp.Services.Models
{
    public class WinerySearchInfo : BaseWineryInfo
    {
        public float Latitude { get; set; }

        public float Longitude { get; set; }
        
        public string Address { get; set; } = string.Empty;

        public string Contact { get; set; } = string.Empty;

        public string Url { get; set; } = string.Empty;
    }
}

File: WineriesApp/src/WineriesApp.UI/src/models/WinerySearchInfo.ts
import { BaseWineryInfo } from "./BaseWineryInfo";

export class WinerySearchInfo extends BaseWineryInfo {
    public latitude: number | undefined;
    
    public longitude: number | undefined;

    public address: string | undefined;

    public contact: string | undefined;

    public url: string | undefined;
}